_developer-content/import-api/guides/sequencing-data-for-the-import-api.md

```diff
@@ -0,0 +1,204 @@
+
+  Stitch uses the `sequence` property on records pushed to the Import API to accomplish this. Every record in a request must have this property.
+
+  In this guide, we'll cover:
+
+  {% for section in page.sections %}
+  - [{{ section.summary }}](#{{ section.anchor }})
+  {% endfor %}
+
+
+# -------------------------- #
+#           CONTENT          #
+# -------------------------- #
+
+sections:
+  - title: "Understanding record sequencing"
+    anchor: "understand-record-sequencing"
+    summary: "The basics of record sequencing"
+    content: |
+      {% for subsection in section.subsections %}
+      - [{{ subsection.title }}](#{{ subsection.anchor }})
+      {% endfor %}
+
+    subsections:
+      - title: "Sequencing and Replication Keys"
+        anchor: "sequence-replication-keys"
+        content: |
+          Think of `sequence` like a [Replication Key]({{ link.replication.rep-keys | prepend: site.baseurl }}).
+
+          For the Import API, it functions similarly to an `updated_at` property: Only records that have a `sequence` value greater than **[TODO- Confirm that this has to be greater than the saved sequence value for the TABLE and not the record. Pretty sure this is the case, but it should be confirmed.]**
+
+          For example: The record below has a `sequence` value of `1574800199000`:
+
+          ```json
+          {
+             "action":"upsert",
+             "sequence":1574800199000,
+             "data":{
+                "id":1,
+                "name":"Finn",
+                "age":15,
+                "has_magic":false
+             }
+          }
+          ```
+
+          In this example, `sequence` is a Unix epoch in milliseconds that translates to `Tuesday, November 26, 2019 8:29:59 PM GMT-05:00`.
+
+      - title: "Record sequencing, Primary Keys, and loading"
+        anchor: "record-sequencing-primary-keys"
+        content: |
+          TODO - Content about Primary Keys and de-duping.
+
+  - title: "Defining record sequences"
+    anchor: "define-record-sequences"
+    summary: "How to define a record's sequence"
+    content: |
+      {% for subsection in section.subsections %}
+      - [{{ subsection.title }}](#{{ subsection.anchor }})
+      {% endfor %}
+
+    subsections:
+      - title: "Generating sequence values"
+        anchor: "generate-sequence-values"
+        content: |
+          When developing your Import API script, you'll need to define how values for the `sequence` property are generated. 
+
+          A simple solution is just to use the current timestamp, but before doing so, consider the following:
+
+          - **How frequently are the records being updated?** Records that are updated every few milliseconds can result in failure if records with identical key values are pushed simultaneously. This means that records with the same Primary Key values cannot be sent during the same clock resolution.
```

Comment:
I could be wrong, but I doubt this would cause the load to fail. I think it would just result in maybe the wrong version of the record being loaded.

Reply:
Ah, perhaps that wording should be changed - I think `failure` here just means `This version of a record won't load successfully`, not that the load itself will fail.